docs/source/reference/integration.rst

```diff
@@ -9,7 +9,7 @@ Integration
 .. autoclass:: ChainerMNStudy
     :members:
 
-.. autoclass:: CmaEsSampler
+.. autoclass:: PyCmaSampler
```

Comment:
Should we keep the previous docs as well, as it's still a public API, just deprecated?